sys/include/net/sixlowpan.h

```diff
@@ -37,112 +37,26 @@ extern "C" {
  *          </a>
  * @{
  */
-#define SIXLOWPAN_UNCOMP            (0x41)      /**< uncompressed 6LoWPAN frame dispatch. */
-#define SIXLOWPAN_FRAG_DISP_MASK    (0xf8)      /**< mask for fragmentation
-                                                 *   dispatch */
-#define SIXLOWPAN_FRAG_1_DISP       (0xc0)      /**< dispatch for 1st fragment */
-#define SIXLOWPAN_FRAG_N_DISP       (0xe0)      /**< dispatch for subsequent
-                                                 *   fragments */
-#define SIXLOWPAN_FRAG_MAX_LEN      (2047)      /**< Maximum datagram size @f$ (2^{11} - 1) @f$ */
 
 /**
  * @brief   Dispatch mask for LOWPAN_IPHC.
  * @see     <a href="https://tools.ietf.org/html/rfc6282#section-3.1">
  *              RFC 6282, section 3.1
  *          </a>
  */
-#define SIXLOWPAN_IPHC1_DISP_MASK   (0xe0)
+#define LOWPAN_IPHC1_DISP_MASK      (0xe0)
```

Comment:
Here and throughout this file: since the name of the file is `sixlowpan.h`, I would keep the name `SIXLOWPAN_`